webdriver/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "wptdashboard-webdriver",
+  "description": "Node packages leveraged by build rules for the wpt.fyi webdriver tests.",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/web-platform-tests/wpt.fyi.git"
+  },
+  "devDependencies": {
```

Comment:
Did you forget to save the dependencies? There's no prod or dev dependencies at all in `package.json`.